cmake/resources/windows.rc.in

```diff
@@ -0,0 +1,34 @@
+#include <winver.h>
+
+IDI_ICON1   ICON    "@RES_DIR@/icon.ico"
+
+VS_VERSION_INFO VERSIONINFO
+    FILEVERSION        @PROJECT_VERSION_MAJOR@,@PROJECT_VERSION_MINOR@,@PROJECT_VERSION_PATCH@,@PROJECT_VERSION_TWEAK@
+    PRODUCTVERSION     @PROJECT_VERSION_MAJOR@,@PROJECT_VERSION_MINOR@,@PROJECT_VERSION_PATCH@,@PROJECT_VERSION_TWEAK@
+    FILEFLAGSMASK      VS_FFI_FILEFLAGSMASK
+    FILEFLAGS          (VS_FF_DEBUG|VS_FF_SPECIALBUILD)
+    FILEOS             VOS__WINDOWS32
+    FILETYPE           VFT_APP
+    FILESUBTYPE        0x0L
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904b0"
+        BEGIN
+            VALUE "ProductName",      "Chatterino"
+            VALUE "ProductVersion",   "@PROJECT_VERSION@"
+            VALUE "CompanyName",      "Chatterino, @PROJECT_HOMEPAGE_URL@"
+            VALUE "FileDescription",  "@PROJECT_DESCRIPTION@"
```

Comment:
The `FileDescription` will show up inside the Task Manager as a group name for chatterino.exe and crashpad_handler.exe.
If you set Chatterino to start with Windows, the application name in the Task Manager Startup tab is also the `FileDescription`.
Maybe it's better to just call it Chatterino or Chatterino + Version.

First your PR, then your PR with changed `FileDescription`.
![image](https://github.com/Chatterino/chatterino2/assets/114905842/e037bc43-bf57-4770-bd4e-a2977f063425)

Reply:
`FileDescription` now includes name and version too
![img](https://github.com/Chatterino/chatterino2/assets/28986062/ba4f9658-94ee-4f7c-b417-1eace9fa0bbe)
i think keeping it more informative than just name+version is better for user, so i kept description too.

By the look on windows services or other popular apps, we are still on shorter end, here's Notepad++ name in taskbar:
![img2](https://github.com/Chatterino/chatterino2/assets/28986062/e31bfa69-376d-448b-8323-b8d28f8473f0)

Reply:
Should we also display `Nightly` on a Nightly build?

Reply:
personally i wouldn't add `Nightly` to FileDescription, feels not needed in context of task manager or file details view. We already have it visible in main window title, where most eyes look.
But if someone thinks otherwise, feel free to discuss it here.

Reply:
I'm fairly sure Notepad++ is the odd one out, from what I can see most applications use their `FileDescription` as a `ProductName` since without one task manager will change the entry based on `File name extensions` being enabled or disabled.

`File name extensions` setting is disabled:
![image](https://github.com/Chatterino/chatterino2/assets/41973452/c2073da7-2627-4d81-a86b-179fd8d7feef)

`File name extensions` setting is enabled:
![image](https://github.com/Chatterino/chatterino2/assets/41973452/c099de90-a708-4835-af20-5ac225d00b59)

(Adam asked in the RL discord what caused it to appear as Runelite.exe and the answer reminded me of this PR)